README.md

```diff
@@ -216,6 +216,7 @@ The letters in brackets indicate:
 | [`Product`](#product)                   |        | ✓      |        |      | n        | Product is the product of all of the elements.  |
 | [`Random`](#random)                     | ✓      | ✓      | ✓      |      | 1        | Random returns a random element by your rand.Source, or zero  |
 | [`Reduce`](#reduce)                     | ✓      | ✓      |        |      | n        | Reduce continually applies the provided function over the slice. Reducing the elements to a single value.  |
+| [`Remove`](#remove)                     | ✓      | ✓      |        | ✓    | n        | Remove returns a new slice that does not include any of the items. |
```

Comment:
Maps should not be ticked.

Did you format this file after running `generate.go`?